zipline/data/us_equity_minutes.py

```diff
@@ -49,10 +48,29 @@ def write_metadata(directory, first_trading_day):
         json.dump(metadata, fp)
 
 
+def _bcolz_minute_index(trading_days):
+    minutes = np.zeros(len(trading_days) * MINUTES_PER_DAY,
+                       dtype='datetime64[ns]')
+    market_opens = tradingcalendar.open_and_closes.market_open
+    mask = market_opens.index.slice_indexer(start=trading_days[0],
```

Comment:
@ssanderson I really like this solution, but there is some contortion needed for consistently getting a dtype of `datetime64` out of the `market_opens` in the trading calendar for the window needed.

Going to leave for now, but may try to get this in soon.